src/test/java/com/google/sps/TextAnalyserTest.java

```diff
@@ -0,0 +1,132 @@
+package com.google.sps.data;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.junit.Test;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+@RunWith(JUnit4.class)
+public final class TextAnalyserTest {
+
+  @Test
+  public void analysePositiveSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("Today is a special day. It is your birthday, so enjoy it!");
+      assertTrue("Sentiment score is not positive.", textAnalyser.getSentimentScore() > 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analyseNegativeSentiment() {
+    try {
+      TextAnalyser textAnalyser = new TextAnalyser("I don't like this.");
+      assertTrue("Sentiment score is not negative.", textAnalyser.getSentimentScore() < 0.0F);
+    } catch (IOException e) {
+      fail(e.getMessage());
+    }
+  }
+
+  @Test
+  public void analysePositiveMood() throws IOException {
```

Comment:
As an opportunity where you can make the tests even better: You can think about how to somehow summarize expected results/behavior in the test name, see e.g. https://qualitycoding.org/unit-test-naming/. There is no single way to name tests, but ideally they state the 1) conditions, 2) what is the subject being tested, and 3) expected results. Example: 1) empty list input, 2) sorting algorithm, 3) empty list as output => "quicksorting_anEmptyList_EmptyList".